core/openapi.yaml

```diff
@@ -37,7 +37,7 @@ paths:
       responses:
         '200':
           $ref: '#/components/responses/LandingPage'
-        '500':
+        default:
```

Comment:
I'm not sure default is ideal here as it would also apply to 2XX and 3XX.

It seems you can use 4XX and 5XX instead of default:
https://swagger.io/docs/specification/describing-responses/

Reply:
Updated to use 4XX and 5XX. Funny thing -- the docs actually give an example of using `default` for the error response, which I think I copied without realizing it wasn't accurate.